dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/dex/DexApiSyntax.scala

```diff
@@ -23,7 +24,23 @@ object DexApiSyntax {
       waitForOrderStatus(order.assetPair, order.id(), status)
 
     def waitForOrderStatus(assetPair: AssetPair, id: Order.Id, status: HttpOrderStatus.Status): F[HttpOrderStatus] =
-      waitForOrder(assetPair, id)(_.status == status)
+      waitForOrder(assetPair, id) { s =>
+
+        def fail() = throw new RuntimeException(s"Expected status $status for order $id, but found ${s.status}")
+        def check(): Boolean = s.status == status
+
+        status match {
+          case Filled | PartiallyFilled => s.status match {
+              case Cancelled | NotFound => fail()
+              case _ => check()
+            }
+          case Accepted | NotFound => s.status match {
```

Comment:
We don't need to match an `expectedStatus` with `NotFound`, because:
* It means there is not such order;
* In `waitForOrderStatus` we are waiting for an order. 
So this method is not applicable for `NotFound`.

Reply:
ok